web/src/features/filters/components/multi-select.tsx

```diff
@@ -118,9 +118,9 @@ export function MultiSelect({
                     >
                       <Check className={cn("h-4 w-4")} />
                     </div>
-                    <span>{option.value}</span>
+                    <span className="overflow-scroll">{option.value}</span>
```

Comment:
```suggestion
                    <span className="overflow-x-scroll">{option.value}</span>
```

lmit this to x in order to prevent weird side-effects?